homeassistant/components/epson/config_flow.py

```diff
@@ -0,0 +1,105 @@
+"""Config flow for epson integration."""
+import logging
+
+import epson_projector as epson
+from epson_projector.const import POWER
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT, CONF_SSL
+from homeassistant.core import callback
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN, TIMEOUT_SCALE
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Optional(CONF_NAME, default=DOMAIN): str,
+        vol.Optional(CONF_PORT, default=80): int,
+        vol.Optional(CONF_SSL, default=False): bool,
+    }
+)
+
+
+async def validate_input(hass: core.HomeAssistant, data):
+    """Validate the user input allows us to connect.
+
+    Data has the keys from DATA_SCHEMA with values provided by the user.
+    """
+    epson_proj = epson.Projector(
+        data[CONF_HOST],
+        websession=async_get_clientsession(hass, verify_ssl=data.get(CONF_SSL, False)),
+        port=data[CONF_PORT],
+    )
+
+    _power = await epson_proj.get_property(POWER)
+
+    if not _power:
+        raise CannotConnect
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for epson."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry):
+        """Epson options flow."""
+        return EpsonOptionsFlowHandler(config_entry)
+
+    async def async_step_user(self, user_input=None):
```

Comment:
Import will be forwarded to `async_step_import` which is missing from config flow